indigo/distribution.yaml

```diff
@@ -10075,6 +10075,22 @@ repositories:
       url: https://github.com/GT-RAIL/rail_collada_models.git
       version: develop
     status: maintained
+  rail_face_detector:
+    doc:
+      type: git
+      url: https://github.com/GT-RAIL/rail_face_detector.git
+      version: master
```

Comment:
quick question: Is it on purpose that the documentation is generated from a different branch than the source code?

Reply:
I plan to move the main code over to master, for now the documentation and code sit on a different branch (once this was published I planned on cleaning up the branch names)